accessibility_api/AccessibilityRequestHandler.py

```diff
@@ -0,0 +1,91 @@
+'''
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this file,
+You can obtain one at http://mozilla.org/MPL/2.0/.
+'''
+
+import sys
+from BaseHTTPServer import BaseHTTPRequestHandler
+from urlparse import urlparse, parse_qs, parse_qsl
+from json import dumps
+from comtypes import CoInitialize
+from accessibility_api.accessibility_lib.scripts.constants import (
```

Comment:
SUCCESS seems to be unused. I think mapping SUCCESS to False and ERROR to True is confusing. I would just get rid of those constants.

In response you can just check if value.get('error') is True directly. Also in case of an error, besides error JSON property that is set to true, it might be useful to have a message property where it is available. Sorry I overlooked that bit yesterday. Otherwise everything looks elegant and clean.